jsonnet/components/prometheus-user-workload.libsonnet

```diff
@@ -142,7 +142,7 @@ function(params)
           // to be immune against UID changes.
           apiGroups: ['security.openshift.io'],
           resources: ['securitycontextconstraints'],
-          resourceNames: ['nonroot'],
+          resourceNames: ['nonroot-v2'],
```

Comment:
maybe update the comment above and explain a bit about why we use `nonroot-v2` rather than `nonroot`?

Reply:
Yes, 100% makes sense, I've added some lines to make it more clear